ui/push-health/MyPushes.jsx

```diff
@@ -60,10 +60,11 @@ class MyPushes extends React.Component {
 
   componentDidUpdate(prevProps) {
     const { user } = this.props;
-
     if (!prevProps.user.isLoggedIn && user.isLoggedIn) {
+      const { author } = parseQueryParams(location.search);
```

Comment:
do you need windows.location.search instead?

Reply:
It's copied and pasted from below, and it works because `location` is global and links (sic) to `document.location`/`window.location`.

Reply:
I saw the javascript linter complain about it

Reply:
get the tests passing and I will review again